retz-client/src/main/java/io/github/retz/web/feign/Server.java

```diff
@@ -0,0 +1,115 @@
+/**
+ *    Retz
+ *    Copyright (C) 2016 Nautilus Technologies, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.github.retz.web.feign;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Objects;
+import java.util.function.Supplier;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+
+import feign.Feign;
+import feign.Param;
+import feign.RequestLine;
+import feign.jackson.JacksonDecoder;
+import feign.jackson.JacksonEncoder;
+import io.github.retz.auth.Authenticator;
+import io.github.retz.protocol.LoadAppRequest;
+import io.github.retz.protocol.Response;
+import io.github.retz.protocol.ScheduleRequest;
+import io.github.retz.protocol.data.Application;
+import io.github.retz.protocol.data.Job;
+
+public interface Server {
```

Comment:
Name like "Retz" or "RetzAPI" would be more suitable as the name of this interface according to [samples](https://github.com/OpenFeign/feign/blob/master/README.md) .

```java
GitHub github = Feign.builder()
                     .encoder(new JacksonEncoder())
                     .decoder(new JacksonDecoder())
                     .target(GitHub.class, "https://api.github.com");
```